app/src/test/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityModelProducerTest.kt

```diff
@@ -141,4 +129,19 @@ class AllPatientsInFacilityUiStateProducerTest {
       assertNotTerminated()
     }
   }
+
+  private fun dispatchScreenCreatedEvent() {
+    MobiusTestFixture<AllPatientsInFacilityModel, AllPatientsInFacilityEvent, AllPatientsInFacilityEffect>(
+        uiEventsSubject,
```

Comment:
Since `uiEvents` is not being used anywhere, we can inline this or make it `Observable.never()`.